CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# dbt_xero_source v0.3.0
+
+## Features
```

Comment:
For each of the main bullets in the CHANGELOG can you include a link to the PR where the change was made. See the recent [dbt_netsuite_source CHANGELOG](https://github.com/fivetran/dbt_netsuite_source/blob/main/CHANGELOG.md) for example. 
![image](https://user-images.githubusercontent.com/74217849/139746989-af9c0dd3-29fc-4aab-a05d-28659120a87a.png)

Reply:
Additionally to the above, can you please add the section at the bottom that just points users to refer to the Github repository release notes for details on the previous releases.